suzieq/sqobjects/basicobj.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, engine_name: str = '',
 
         if engine_name and engine_name != '':
             self.engine = get_sqengine(engine_name, self._table)(self)
-        elif self.ctxt.engine:
+        elif self.ctxt.cfg.get('ux', {}).get('engine', 'pandas'):
             self.engine = get_sqengine(self.ctxt.engine, self._table)(self)
```

Comment:
I think here you are always using pandas, even if in the config there is `rest`. That's because you are passing `self.ctxt.engine`, which is always set to pandas by the[ post_init of the context](https://github.com/netenglabs/suzieq/blob/develop/suzieq/shared/context.py#L30), unless explicitly set, and this only set by the CLI. Which means that in all the other cases we still go on using pandas even though the engine in the conf file is rest.
However I'm not sure this solution will work, even if you pass the right engine, then, you won't get all the parameters needed to connect to the rest server (address, port and api key), so I think we should move what is [here](https://github.com/netenglabs/suzieq/blob/develop/suzieq/cli/sq_nubia_context.py#L41) in the post_init of the context. But this is not enough, because we need to prevent the rest server from using the rest engine.

Reply:
Sigh. I could've sworn I tested the code. Anyways, you're right. Fixed it.